pkg/kubelet/dockershim/docker_container.go

```diff
@@ -131,6 +131,10 @@ func (ds *dockerService) CreateContainer(podSandboxID string, config *runtimeapi
 			OpenStdin: config.Stdin,
 			StdinOnce: config.StdinOnce,
 			Tty:       config.Tty,
+			// Disable Docker's health check until we officially support it.
+			Healthcheck: &dockercontainer.HealthConfig{
```

Comment:
This is the compatibility between new docker client API and old docker daemon, which should not be an issue?